examples/test/test_bert_cuda_gpu.py

```diff
@@ -0,0 +1,35 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import os
+from pathlib import Path
+
+import pytest
+from utils import check_search_output, patch_config
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup():
+    """setup any state specific to the execution of the given module."""
+    cur_dir = Path(__file__).resolve().parent.parent
+    example_dir = cur_dir / "bert"
+    os.chdir(example_dir)
+    yield
+    os.chdir(cur_dir)
+
+
+@pytest.mark.parametrize("search_algorithm", ["tpe"])
+@pytest.mark.parametrize("execution_order", ["joint"])
+@pytest.mark.parametrize("system", ["aml_system"])
+@pytest.mark.parametrize("olive_json", ["bert_cuda_gpu.json"])
+@pytest.mark.parametrize("enable_cuda_graph", [True, False])
```

Comment:
Do we need test perf_tuning config here? It may be like something in perf_tuning pass tests.

Reply:
We need to submit e2e aml job to run test on gpu machine

Reply:
Yes, I mean maybe we can just keep one option(like enable_cuda_graph=True). Then it will save half time to run tests.